_distro_map.yml

```diff
@@ -24,6 +24,9 @@ openshift-origin:
     enterprise-4.10:
       name: '4.10'
       dir: '4.10'
+    enterprise-4.10:
```

Comment:
@mburke5678 Should this have been `enterprise-4.11` ?

Reply:
@bergerhoffer Yes. I merged this PR in error and meant to come back and fix it. Since it was only a cosmetic issue, I decided not to revert.  I see you fixed it.  Thanks!